workflows/cloudwatch_logs_big_query/resources/put-log-events.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+FILES=$(ls file*.json)
+
+for f in $FILES; do
+    echo "$f"
+    aws logs put-log-events --log-group-name "/workflows/cloudwatch-log/big-query" --log-stream-name stream1 --log-events file://"$f" --no-cli-pager
```

Comment:
I had to remove the `--no-cli-pager` flag for the script to run successfully.
```suggestion
    aws logs put-log-events --log-group-name "/workflows/cloudwatch-log/big-query" --log-stream-name stream1 --log-events file://"$f"
```

Reply:
Are you running the latest CLI? This should work fine.